functionary/core/utils/tasking.py

```diff
@@ -179,12 +179,13 @@ def _handle_file_parameters(task: Task) -> None:
     """
     environment = task.environment
     parameters = task.parameters
-    schema: dict = task.function.schema
 
-    param_map = parameter_mapping(schema)
-    for param_name, param_meta in param_map.items():
-        param_formats = [_format for _, _format in param_meta]
-        if "uri" in param_formats:
+    for parameter in task.function.parameters.filter(
+        parameter_type=PARAMETER_TYPE.FILE
```

Comment:
Yeah, that was indeed doable.  Added that filter and then got rid of the subsequent check since it's now only operating on file parameters that are definitely in the task parameters.